scripts/netContain/contivd.sh

```diff
@@ -1,79 +1,76 @@
 #!/usr/bin/bash
```

Comment:
This script is only modified to format the script properly, to remove trailing whitespace and follow best practices.